free-2.md

```diff
@@ -1,44 +1,85 @@
 # Free project 2
 
+MarkTeX?? : Simple and possible to use in a WYSIWYG editor like Markdown,
+but targeted at LaTeX's domain.
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
 
+
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+LaTeX is way too complicated. And moreover, all the WYSIWYG attempts suck.
+What we need is a language that is as simple as possible but can still accomplish
+similar tasks.
+Moreover, we need a language with separated concerns. I don't want to use the same language 
+or process to construct my document that I use to make a new character glyph or do
+calculations.
+I don't even want to describe the specifics of styles in the same way.
```

Comment:
If you want them to be separate, why don't you just use LaTeX and something like google docs for different purposes?